bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/install/IDFDownloadPage.java

```diff
@@ -397,9 +397,10 @@ private void validate()
 			}
 			
 			String idfPyPath = idfPath + File.separator + "tools" + File.separator + "idf.py"; //$NON-NLS-1$ //$NON-NLS-2$
-			if (!new File (idfPyPath).exists())
+			String idfToolsPyPth = idfPath + File.separator + "tools" + File.separator + "idf_tools.py"; //$NON-NLS-1$ //$NON-NLS-2$
+			if (!new File (idfPyPath).exists() && !new File (idfToolsPyPth).exists())
```

Comment:
We need to show an error message even if one of the files does not exist.

```
if (!new File(idfPyPath).exists() || !new File(idfToolsPyPath).exists())
{
    // show error message
}
```

Reply:
@kolipakakondal Thanks for the feedback but we are already setting error message on wizard do you think we should show another popup message to user on an already opened popup wizard?

Reply:
@kolipakakondal Done